decko-rails/lib/decko/rails.rb

```diff
@@ -0,0 +1,15 @@
+DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
+
+module Decko
+  module Rails
```

Comment:
Oh, then '-' in a name implied '/' in the require name?  I didn't completely understand that.